### PR TITLE
Update dashboard client API doc

### PR DESCRIPTION
**Pull request by julien-nc**
* Mention the new endpoint to list enabled widgets
* Give response examples for both client-related endpoints

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Will there also be docs for the Nextcloud app devs, not just the client devs?

File: developer_manual/digging_deeper/dashboard.rst [hunk 1]
```diff
@@ -219,7 +219,63 @@ This interface contains an extra `getItems` method which returns an array of `OC
...
+
+.. code-block:: bash
+
+    curl -u user:passwd https://my.nextcloud.org/ocs/v2.php/apps/dashboard/api/v1/widgets \
+        -H "Accept: application/json" \
+        -X GET
+
+Example response:
+
+.. code-block:: bash
```
Comment (CarlSchwan) [inline developer_manual/digging_deeper/dashboard.rst:232]:
```suggestion
.. code-block:: json
```

File: developer_manual/digging_deeper/dashboard.rst [hunk 2]
```diff
@@ -230,3 +286,58 @@ From the client point of view, the dashboard widget items can then be obtained w
 
 If your client periodically gets widget items content with this API,
 include the latest `sinceId` for each widget to avoid getting the items you already have.
+
+Example response:
+
+.. code-block:: bash
```
Comment (CarlSchwan) [inline developer_manual/digging_deeper/dashboard.rst:292]:
```suggestion
.. code-block:: json
```

**Comment by julien-nc** [comment]
> Will there also be docs for the Nextcloud app devs, not just the client devs?

@ChristophWurst Done :grin:
It now mentions all new interfaces and provides usage examples + example responses.

**Pull request merged**